app/src/main/java/com/github/gotify/service/WebSocketService.java

```diff
@@ -314,6 +315,19 @@ private void showNotification(
         b.setContentText(message);
         b.setStyle(new NotificationCompat.BigTextStyle().bigText(formattedMessage));
 
+        String notificationImageUrl =
+                Extras.getNestedValue(String.class, extras, "client::notification", "imageUrl");
```

Comment:
I renamed this to bigImageUrl because I'd expect that imageUrl would override the largeIcon on the notification.

![image](https://user-images.githubusercontent.com/14895212/144478731-2d3d12cd-39d5-42ed-bb0c-9d1654136799.png)